src/main/kotlin/no/nav/familie/ba/sak/behandling/steg/StegService.kt

```diff
@@ -113,42 +114,40 @@ class StegService(
                 hentBehandlingSteg(StegType.FERDIGSTILLE_BEHANDLING) as FerdigstillBehandlingSteg
 
         return håndterSteg(behandling, behandlingStegSteg) {
-            behandlingStegSteg.utførSteg(behandling, "")
+            behandlingStegSteg.utførStegOgAngiNeste(behandling, "")
         }
     }
 
     // Generelle stegmetoder
-    fun håndterSteg(behandling: Behandling, behandlingSteg: BehandlingSteg<*>, uførendeSteg: () -> Behandling): Behandling {
+    private fun håndterSteg(behandling: Behandling, behandlingSteg: BehandlingSteg<*>, utførendeSteg: () -> StegType): Behandling {
         try {
             if (behandling.steg == sisteSteg) {
                 error("Behandlingen er avsluttet og stegprosessen kan ikke gjenåpnes")
             }
 
             if (behandlingSteg.stegType().rekkefølge > behandling.steg.rekkefølge) {
-                error("${SikkerhetContext.hentSaksbehandler()} prøver å utføre steg ${behandlingSteg.stegType()}," +
+                error("${SikkerhetContext.hentSaksbehandlerNavn()} prøver å utføre steg ${behandlingSteg.stegType()}," +
                       " men behandlingen er på steg ${behandling.steg}")
             }
 
             val behandlerRolle =
                     SikkerhetContext.hentBehandlerRolleForSteg(rolleConfig, behandling.steg.tillattFor.minBy { it.nivå })
 
-            LOG.info("${SikkerhetContext.hentSaksbehandler()} håndterer ${behandlingSteg.stegType()} på behandling ${behandling.id}")
+            LOG.info("${SikkerhetContext.hentSaksbehandlerNavn()} håndterer ${behandlingSteg.stegType()} på behandling ${behandling.id}")
             if (!behandling.steg.tillattFor.contains(behandlerRolle)) {
-                error("${SikkerhetContext.hentSaksbehandler()} kan ikke utføre steg '${behandlingSteg.stegType()}")
+                error("${SikkerhetContext.hentSaksbehandlerNavn()} kan ikke utføre steg '${behandlingSteg.stegType()} pga manglende rolle.")
```

Comment:
```suggestion
                error("${SikkerhetContext.hentSaksbehandlerNavn()} kan ikke utføre steg '${behandlingSteg.stegType()}' pga manglende rolle.")
```

Skulle vi lagt til displayName på stegene? Sikkert ikke like pent for SB å få en enum slengt i fjeset :P

Reply:
Det vil nok være en klar forbedring ja :) Jeg fikser det.

Reply:
Vi har allerede et felt "beskrivelse" på stegene som fint kunne vært brukt hadde det ikke vært for det siste steget der det er lagt på litt ekstra informasjon som gjør det uegnet som displayName. Det står det "Behandlingen er avsluttet og kan ikke gjenåpnes". Kunne vi redusert dette til kun "Behandling avsluttet" så slipper vi å legge til enda et nytt felt som i nesten alle tilfeller vil være identisk? Ser at beskrivelse idag kun brukes et sted i koden, for en teller-metrikk. Hva mener du @henninghaakonsen ?

Reply:
Støtter det forslaget 👍